exo-go/src/process_helpers/index.go

```diff
@@ -0,0 +1,57 @@
+package processHelpers
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+	"time"
+
+	shellwords "github.com/mattn/go-shellwords"
+)
+
+// Run runs the given command, waits for the process to finish and
+// returns the output string and error (if any)
+func Run(command string) (string, error) {
```

Comment:
These functions and the relevant code will be updated to [this](https://github.com/Originate/exosphere/blob/1f77a17ca4e1a3fd7bbb1663563734bf52dbf719/exo-go/src/process_helpers/index.go) once #322 is merged.